frontend/public/components/dashboard/dashboards-page/cluster-dashboard/status-card.tsx

```diff
@@ -151,10 +151,28 @@ export const StatusCard = connect<StatusCardProps>(mapStateToProps)(({ k8sModels
     DynamicDashboardsOverviewHealthSubsystem
   >(isDynamicDashboardsOverviewHealthSubsystem);
 
-  const subsystems = React.useMemo(
-    () => filterSubsystems([...subsystemExtensions, ...dynamicSubsystemExtensions], k8sModels),
-    [subsystemExtensions, dynamicSubsystemExtensions, k8sModels],
-  );
+  const subsystems = React.useMemo(() => {
+    const dynamicExtensions: any = dynamicSubsystemExtensions.map((e) => {
```

Comment:
Yeah. my bad. I was being lazy. I moved code around a bit so there is no more `any`

FIY this solves a bug where `PrometheusHealthItem` expects that `popupComponent` is async loaded but we are actually passing already resolved popup (because we use `useResolvedExtensions`). So I had to wrap the resolved `popupComponent` into `() => Promise.resolve`.

The other option is to use just `useExtensions` but then we have other CodeRefs like `healthHandler` or `additionalResource` which would mean changing a lot more code

Reply:
> The other option is to use just `useExtensions` but then we have other CodeRefs like `healthHandler` or `additionalResource` which would mean changing a lot more code

@rawagner @christianvogt FYI - we might need to add an API for selective CodeRef value resolution into [openshift/dynamic-plugin-sdk](https://github.com/openshift/dynamic-plugin-sdk) which is currently under development.